src/index.ts

```diff
@@ -210,7 +211,10 @@ export function createAlchemyWeb3(
         params: [
           {
             ...params,
-            maxCount: params.maxCount ? toHex(params.maxCount) : undefined,
+            maxCount:
+              params.maxCount !== undefined
```

Comment:
This is a personal style thing, but I prefer doing null checks using `== null` or `!= null`. Using the double-equals comparison to `null` means that it checks for both `null` and `undefined`, which is nice a lot of the time where you'd want to treat them the same. This is the only time I ever use `==` instead of `===`.

If you think that's too tricky, then that's fine too and you can leave as is.